.github/workflows/go.yml

```diff
@@ -14,7 +14,9 @@ jobs:
     steps:
       - name: install ignite
         run: |
-          curl https://get.ignite.com/cli! | bash
+          wget https://github.com/ignite/cli/releases/download/v0.27.1/ignite_0.27.1_linux_amd64.tar.gz
```

Comment:
Also pins to the version we currently use locally.

Reply:
Can you #PUC that this pins the version and we can use `curl https://get.ignite.com/cli! | bash` (when it works) for the latest & greatest?